test/functional/legacy_editions_controller_test.rb

```diff
@@ -1304,4 +1304,55 @@ class LegacyEditionsControllerTest < ActionController::TestCase
       end
     end
   end
+
+  context "when 'restrict_access_by_org' feature toggle is disabled" do
+    setup do
+      test_strategy = Flipflop::FeatureSet.current.test!
```

Comment:
same as above, We can move the test_strategy to the top most setup block of the class, avoids repeating.